_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Swipe down from the video details to minimize. This gives you the cutest little video, from which you can play, pause, or close down this little rascal pulling on your heartstrings. Browse away while the video stays put, or carries on. Without distractions like these, how do you really know the video you are watching is the best possible distraction from something else? In widescreen distraction, you can swipe down on the title to minimize. Other swipage in other areas still controls volume, brightness and gestures.
 <video src="/img/landscape_swipe_down.webm" poster="/img/0.20.0_thumbnail_swipe_down.png" class="img-responsive" autoplay loop muted controls></video>
-Since the player can now be minimized instead of closed, this means you get to retain your opened video queue! Let us go through an example: You watch a video, minimize it, then browse the trending page (you insane person), watch a video from there, quickly minimize it, browse for some calming music, and finally watch a video about pandas. As long as the minimized player isn't closed (by using the cross button or swiping down on it), you can tap back to go through those videos in reverse order! This allows you to browse a whole bunch of videos and quickly build up a video queue before you start watching them. You were able do this earlier, but you had to use the background or popup player queues as a workaround, since the main player's queue isn't accessible outside of the player.
+Minimizing the player instead of closing it means retaining your place in the video bread queue!
+Example: Be supposed to be doing something else, watch a video instead, minimize it, then browse _the trending page_ (you insane person). Watch some filth from there, then quickly minimize it, and shudder. Now browse for some calming music, and finally watch a video about pandas.
+
+If the player isn't closed (by using the cross button or swiping down on it), and instead stays minimized, you can tap "Back" to go through those videos in reverse order! You are making history, as the lead role of your own exploration. Pick videos for a quickly built video queue before watching.
+Before this was only possible by using the background or pop-up player.
 
 ### Thumbnail progress bar
 
-You can only see this when you play the video in the background or pop-up, but there is now a shiny new progress bar underneath the thumbnail! It updates in real time, gets a little counter and a red bar - it's great, really.  
```

Comment:
Why do I never know what is being communicated, without keeping prior material in mind for no good reason. Keep in the story, don't make spaghetti. Last sentence passable. The format is like a video though. Establish one voice, make it work.

Reply:
> without keeping prior material in mind

What prior material? The sentence seems self-contained to me.

Reply:
The whatever relates to "this". I still don't know what it is. Is it the thumbnail? Is it the progress bar? Now there is another "it" to clear that up? Clear as mud! Is it the progress bar underneath, or is it the thumbnail. Lets get the boolean logic out to find out what this means. "a little counter and a red bar" is the help you get.